app/assets/javascripts/backbone/views/calendar_view.coffee

```diff
@@ -29,6 +29,12 @@ class MS.Views.CalendarView extends Backbone.View
   # Load custom content inside the fullCalendar rendered event rather than title only
   eventAfterRender: (calEvent, initialElement) ->
     $(initialElement).find('.fc-content').append(calEvent.html)
+
+    if calEvent.is_finalized
+      $(initialElement).addClass('finalized')
+    else
+      $(initialElement).addClass('draft')
```

Comment:
Given that CalendarEventSerializer already has the event_classes method, perhaps better to put this logic in there?

Reply:
I tried, but was having trouble with it working, so this was the best solution I could come up with. I'm not sure why it was not working.

Reply:
I figured out what I needed to change. Fixed. Thanks!

Reply:
Update: The problem is that the event serializer does not successfully distinguish between finalized and draft steps. I'm not sure why that happens. But finalized steps get the `.draft` class though they are not drafts.